### PR TITLE
fix: Record submission error message position

### LINKED ISSUE #2531: Record submission error msg improvement.
When you submit a proposal and it results in some type of error, there's no longer any indication that an error occured because of where the error message placement is.  I have to scroll up to see that there was an error.  This is a confusing UX.

image

image

I think the way to improve this is to move the error message banner to below the submit button.

### PR DESCRIPTION
**Pull request by thi4go**
This diff moves the error message position to below the Submit button on the record submission page.

closes #2531 

UI changes:

Desktop:
image

Mobile (iPhone X):
image

### CODE REVIEW

File: src/components/ProposalForm/ProposalForm.jsx [hunk 1]
```diff
@@ -399,6 +398,16 @@ const ProposalForm = React.memo(function ProposalForm({
           </Row>
         </>
       )}
+      {hasError && (
+        <Row>
+          <Message
+            id="errorMessage"
```
Comment (tiagoalvesdulce) [inline src/components/ProposalForm/ProposalForm.jsx:404]: can you give a more detailed id? eg: `record-submission-error-message`

**Review by tiagoalvesdulce — CHANGES_REQUESTED** [review]
Left one inline comment

**Review by victorgcramos — APPROVED** [review]
ACK!

**Review by lukebp — APPROVED** [review]
ACK

File: src/components/ProposalForm/ProposalForm.jsx (hunk 1 above)
Comment (thi4go) [reply src/components/ProposalForm/ProposalForm.jsx]: addressed mate 👍

**Comment by thi4go** [comment]
@tiagoalvesdulce addressed the comment mate, ready for review 👍

**Review by tiagoalvesdulce — APPROVED** [review]
LGTM

**Pull request merged**